directanswercards/documentsearch-standard/component.js

```diff
@@ -54,6 +59,27 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
   static defaultTemplateName (config) {
     return 'directanswercards/documentsearch-standard';
   }
+
+  /**
+   * construct a URL that links to a specific portion of a page, using a text snippet provided in the URL.
+   * @param {Object} snippet the snippet for the document search direct answer
+   * @param {string} baseUrl website or landingPageURL from the entity related to the snippet
+   * @returns a URL with text fragment URI component attached
+   */
+  getUrlWithTextHighlight(snippet, baseUrl) {
```

Comment:
should this be a formatter?

Reply:
hmm I guess this could be used elsewhere to format url. I will move it